samples/boards/stm32/power_mgmt/blinky/boards/b_u585i_iot02a.overlay

```diff
@@ -0,0 +1,10 @@
+ /*
+  * give a prescaler to the lptim clock. For the stm32U5 a pre-div factor of 2
+  * applied to the lptim clock source. So a prescaler of 16 gives LSE/2 / 16 = 1024Hz
+  * so that the sleep period is of 64s in the sample application
+  * with a LSE clock div by <1> to <8>, CONFIG_SYS_CLOCK_TICKS_PER_SEC is 4096
+  * with a LSE clock div by >= <16>, CONFIG_SYS_CLOCK_TICKS_PER_SEC is LSE / prescaler
+  */
+&stm32_lp_tick_source {
+	st,prescaler = <16>;
+};
```

Comment:
Shoudn't this be removed since the static prescaler has been removed in #65683 ?

Reply:
We can have a sample application running on this target with lptim at 2048Hz and on another platform with lptim clock at 1024Hz. Sleeping period of 2000ms is unchanged